plugins/utils/repeat.nix

```diff
@@ -0,0 +1,17 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "repeat";
+  originalName = "vim-repeat";
+  defaultPackage = pkgs.vimPlugins.vim-repeat;
```

Comment:
We can use `hasSettings` to prevent `mkNeovimPluin` from including a `settings` option.

Reply:
However since this is a vim plugin, not a lua plugin, we should probably be using `mkVimPlugin` instead of `mkNeovimPlugin`.

If we don't set `globalPrefix` then no settings option will be added, however if the plugin has some globals-based config, we could set `globalPrefix = "repeat_"` and a freeform settings option will be created that maps to prefixed vim globals.

Reply:
This plugin has no settings/option whatsoever.